src/com/projectkorra/projectkorra/firebending/lightning/Lightning.java

```diff
@@ -83,11 +116,15 @@ public Lightning(final Player player) {
 			if (!getAbility(player, Lightning.class).isCharged()) {
 				return;
 			}
+			if (getAbility(player, Lightning.class).state == State.CHAIN) {
+				getAbility(player, Lightning.class).remove();
+			}

```

Comment:
Doesn't this mean that instantly starting to charge another one will cancel the existing lightning?

Reply:
when the existing instance is chaining, yeah. i could remove this

Reply:
I think that it makes sense to have a currently chaining lightning arcing around while still preparing a new one. It leads to some interesting mechanics and makes sense design-wise.